docs/astro/sql/operators/export.rst

```diff
@@ -0,0 +1,30 @@
+======================================
```

Comment:
```suggestion
.. _export_file:

======================================
```

Reply:
Will be useful to for other operator to refer

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/633/commits/58f407bd813492247e2bc4bc31c010df06a072df)